### ISSUE TITLE
Playing YouTube videos

### ISSUE THREAD

**Issue by DeepuV** [newshorts/InlineVideo#2]
Hi is there any way to play YouTube video using this library? 
Thanks in Advance..

**Comment by fregante** [comment]
No, with YouTube videos you don't direct access to the `<video>` element so you can't _fix_ it
